<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigMenu</class>
 <widget class="QWidget" name="ConfigMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1044</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_build">
      <attribute name="title">
       <string>Build Settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_16">
         <property name="text">
          <string>Build Tools</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="0" column="1">
           <widget class="QLineEdit" name="cc"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="toolTip">
             <string>%{CC}</string>
            </property>
            <property name="text">
             <string>C Compiler</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="ld"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="toolTip">
             <string>%{LD}</string>
            </property>
            <property name="text">
             <string>Linker</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_2">
            <property name="toolTip">
             <string>%{FIX}</string>
            </property>
            <property name="text">
             <string>GBA Fix</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="fix"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="objcopy"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="toolTip">
             <string>%{OBJCOPY}</string>
            </property>
            <property name="text">
             <string>Objcopy</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Custom Script</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="custom"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_17">
         <property name="text">
          <string>Build Options</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout_3">
          <item row="1" column="0">
           <widget class="QLabel" name="label_9">
            <property name="toolTip">
             <string>%{INCLUDES}</string>
            </property>
            <property name="text">
             <string>Includes</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="toolTip">
             <string>%{LIBS}</string>
            </property>
            <property name="text">
             <string>Libraries</string>
            </property>
           </widget>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_4">
            <property name="toolTip">
             <string>Target hardware %{ARCH}</string>
            </property>
            <property name="text">
             <string>Architecture</string>
            </property>
           </widget>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="arch"/>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="label_11">
            <property name="toolTip">
             <string>Linker flags %{LDFLAGS}</string>
            </property>
            <property name="text">
             <string>Linker Flags</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_7">
            <property name="toolTip">
             <string>Compiler flags %{CFLAGS}</string>
            </property>
            <property name="text">
             <string>C Flags</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QLineEdit" name="ldflags"/>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="cflags"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="toolTip">
             <string>Target ROM %{ROM}</string>
            </property>
            <property name="text">
             <string>ROM File</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="rom"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="includes"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="libs"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_18">
         <property name="text">
          <string>Build Steps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string/>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="1">
           <widget class="QLineEdit" name="compile_step"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="toolTip">
             <string>Recipe for compiling %{SOURCE} to %{OBJECT}. Builds the %{OBJECTS} list</string>
            </property>
            <property name="text">
             <string>Compile</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_13">
            <property name="toolTip">
             <string>Recipe for linking %{OBJECTS} to an ELF.</string>
            </property>
            <property name="text">
             <string>Link</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_14">
            <property name="toolTip">
             <string>Recipe to strip the ELF binary to a GBA ROM</string>
            </property>
            <property name="text">
             <string>Objcopy</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_15">
            <property name="toolTip">
             <string>Recipe to fix the GBA ROM header</string>
            </property>
            <property name="text">
             <string>Fixup</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="link_step"/>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="objcopy_step"/>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="fix_step"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="reset_defaults">
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Reset Defaults</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
